### PR TITLE
Update amp-sidebar.md with UX considerations

### PR DESCRIPTION
**Pull request by sriramkrish85**
CC @Meggin @rudygalfi @ericlindley-g

### CODE REVIEW

File: extensions/amp-sidebar/amp-sidebar.md [hunk 1]
```diff
@@ -106,6 +106,12 @@ The `amp-sidebar` component can be styled with standard CSS.
 - The height of the `amp-sidebar` may be set to adjust the height of the sidebar if required. If the height exceeds 100vw then the sidebar will have a vertical scrollbar. The preset height of the sidebar is 100vw and can be overridden in CSS to make it shorter.
 - The current state of the sidebar is exposed via the `open` attribute that is set on the `amp-sidebar` tag when the side bar is open on the page.
 
+##UX Considerations
```
Comment (cramforce) [inline extensions/amp-sidebar/amp-sidebar.md]: nit: space after `##`

Comment (cramforce) [inline extensions/amp-sidebar/amp-sidebar.md]: lower case `C`

**Comment by cramforce** [comment]
LGTM

File: extensions/amp-sidebar/amp-sidebar.md (hunk 1 above)
Comment (sriramkrish85) [inline extensions/amp-sidebar/amp-sidebar.md]: Done

**Pull request merged**